sechub-web-ui-solution/helm/sechub-web-ui/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: MIT
+
+apiVersion: v2
+name: sechub-web-ui
+description: SecHub Web UI Helm chart for Kubernetes to deploy a vue.js web application
+
+type: application
+
+maintainers:
```

Comment:
We have removed the "maintainers" section in our other Helm charts.
I suggest to do the same here...
```suggestion
```